aiida/common/profile.py

```diff
@@ -126,6 +126,28 @@ def rmq_prefix(self):
     def is_test_profile(self):
         return self._test_profile
 
+    @property
+    def configured_user_email(self):
+        """
+        Return the email (that is used as the username) configured during the
+        first verdi install.
+
+        :return: the currently configured user email address
+        :rtype: str
+        """
+        from aiida.common.setup import get_profile_config, DEFAULT_USER_CONFIG_FIELD
+        from aiida.backends import settings
+
+        try:
+            email = self.config[DEFAULT_USER_CONFIG_FIELD]
+        # I do not catch the error in case of missing configuration, because
+        # it is already a ConfigurationError
+        except KeyError:
```

Comment:
Was this an exception before as well?
Should it be forbidden not to have a default user?

Reply:
Yes, it's forbidden.  Can't store any nodes without one.